utils/moduleVisitor.js

```diff
@@ -91,12 +91,15 @@ exports.default = function visitModules(visitor, options) {
   }
 
   if (options.commonjs || options.amd) {
+    const currentCallExpression = visitors['CallExpression']
     visitors['CallExpression'] = function (call) {
```

Comment:
i know this predates your PR, but:
```suggestion
    visitors.CallExpression = (call) => {
```

Reply:
I agree, but this seems to have done throughout the project.

See:
- [`src/rules/no-commonjs`](https://github.com/benmosher/eslint-plugin-import/blob/master/src/rules/no-commonjs.js#L88)
- [`src/rules/default`](https://github.com/benmosher/eslint-plugin-import/blob/master/src/rules/default.js#L36)

If we do it for this case, shouldn't we do it for all of them?